Explorer/Assets/DCL/Chat/ChatMessagesBus.cs

```diff
@@ -25,15 +25,10 @@ private void CreateTestChatEntry()
         }
 
         //Add message will be called from the message handling system of the livekit integration
-        public void AddMessage()
-        {
-            OnMessageAdded?.Invoke(ProcessChatMessage());
-        }
 
-        //This function will get the message from the livekit integration and convert it to a ChatMessage
-        private ChatMessage ProcessChatMessage()
+        public void Send(string message)
         {
-            return new ChatMessage();
+            OnMessageAdded?.Invoke(new ChatMessage(message, "Self", Random.Range(0, 2) == 0 ? "" : "#asd38", true));
```

Comment:
I've seen `#asd38` being used twice, if this is intended to be a placeholder for something and that something fails, people are going to see messages from `#asd38`? If that's it then I would change the name to #unknown or something like that.